pkg/indicator/gma.go

```diff
@@ -0,0 +1,72 @@
+package indicator
+
+import (
+	"math"
+
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+// Geometric Moving Average
+//go:generate callbackgen -type GMA
+type GMA struct {
```

Comment:
you can add GMA to the standard indicator struct~

Reply:
I'll keep it here, since there's no standard implementation/definition of GMA in any other platform/lib.